packages/extension-ui/src/Popup/Accounts/Account.tsx

```diff
@@ -32,11 +49,19 @@ function Account ({ address, className, isExternal }: Props): React.ReactElement
     _toggleEdit();
   };
 
-  return (
+  return <div className={className}>
```

Comment:
I think the `className` is not used here anymore.

Reply:
It's used for passing styled-component styles